examples/rh-sso/client-registrations/redis_pool.lua

```diff
@@ -0,0 +1,28 @@
+local redis = require 'resty.redis'
+
+-- Redis connection parameters
+local _M = {
+  host      = os.getenv("REDIS_HOST") or '127.0.0.1',
+  port      = 6379,
```

Comment:
Yes. Then it can be reused in `threescale_utils`, and also in [apicast-xc](https://github.com/3scale/apicast-xc/blob/master/xc/redis_pool.lua)